src/utils.ts

```diff
@@ -151,6 +151,8 @@ export function isWhitelisted(url: string) {
     'https://prod.doppleracademy.com/',
     'https://doppleracademy.com/',
     'http://doppleracademy.com/',
+    'https://app2.bancoprovinciamail.com.ar/',
+    'https://app2.fromdoppler.com/',
```

Comment:
If we change all our redirections to include the domain, I think that you will need to also add test environments: 
* `http://localhost/`, `http://localhost:` (I am not sure about the ports)
* `https://localhost/`, `https://localhost:` (I am not sure about the ports)
* `http://localhost` (I am not sure about the ports)
* `https://appint.fromdoppler.com/` 
* `https://appqa.fromdoppler.com/`
* https?

Reply:
I added `http://localhost` and `https://localhost` to allow more freedom (avoiding port)

Reply:
In that way, you are also allowing `http://localhost.malisioussite.com`